wallet/src/de/schildbach/wallet/ui/dashpay/CreateIdentityService.kt

```
@@ -532,14 +534,14 @@ class CreateIdentityService : LifecycleService() {
                 blockchainIdentityData = BlockchainIdentityData(IdentityCreationState.NONE,
                         null, username, usernameSecondary, null, false,
                         usingInvite = true, invite = invite)
-                platformRepo.updateBlockchainIdentityData(blockchainIdentityData)
+                identityRepository.updateBlockchainIdentityData(blockchainIdentityData)
             }
             else -> {
                 throw IllegalStateException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)